packages/sit-onyx/src/styles/mixins/input.scss

```diff
@@ -96,6 +96,17 @@
     }
   }
 
+  &__error-message {
+    display: none;
+  }
+  &:has(#{$base-selector}__native:user-invalid) {
+    #{$base-selector}__error-message {
+      display: flex;
+      flex-flow: row nowrap;
```

Comment:
```suggestion
```

Thats the default for flex :)

Reply:
oops I thought wrap was default 🤦🏻‍♀️ thx